forte/processors/misc/__init__.py

```diff
@@ -18,4 +18,3 @@
 from forte.processors.misc.vocabulary_processor import *
 from forte.processors.misc.delete_overlap_entries import *
 from forte.processors.misc.simple_processors import *
-from forte.processors.misc.remote_processor import *
```

Comment:
This might break some code in `stave` (e.g., the [remote handler](https://github.com/mylibrar/stave/blob/master/simple-backend/stave_backend/handlers/nlp.py#L20) used by eliza project). I guess we can put the try-import block of `requests` under `RemoteProcessor.__init__ ` and revert update here and in `remote_processor_test.py`.